python/lsst/__init__.py

```diff
@@ -0,0 +1,20 @@
+# This file is part of sdm_tools.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
```

Comment:
For consistency with other pipeline packages please add the 

```python
import pkgutil

__path__ = pkgutil.extend_path(__path__, __name__)
```
here and in the next directory down.

Reply:
This is not needed in `lsst/sdm/tools`?

Reply:
you aren't extending anything in the lsst/sdm/tools directory. That has your code in it.